### ISSUE TITLE
Fails to build

### ISSUE THREAD

**Issue by jacek99** [upsidetravel/bucket-antivirus-function#26]
Checked out on Fedora 27, run make for the first time....
```
make
rm -rf compile/lambda.zip
docker run --rm -ti \
	-v /home/jfurmank/src/tmp/bucket-antivirus-function:/opt/app \
	amazonlinux:latest \
	/bin/bash -c "cd /opt/app && ./build_lambda.sh"
/bin/bash: ./build_lambda.sh: Permission denied
make: *** [Makefile:34: archive] Error 126
```

**Comment by jonmelia** [comment]
I've had this problem when I tried to build it on Centos.  In the end I span up an AWS Linux EC2 instance and ran it on there where it worked.

**Comment by nreddipalle** [comment]
I am getting the below error when running on an AWS Linux instance... 
![image](https://user-images.githubusercontent.com/28302399/42055334-d4c61506-7ae4-11e8-9988-a2d7e00bb335.png)

**Comment by denniswebb** [comment]
Fixed by #52
